Neos.Flow/Classes/Mvc/ActionResponse.php

```diff
@@ -0,0 +1,104 @@
+<?php
+namespace Neos\Flow\Mvc;
+
+use Neos\Flow\Annotations as Flow;
+use Psr\Http\Message\UriInterface;
+
+/**
+ * The new minimal MVC response object.
```

Comment:
nitpick: "new" is relative, especially in code that is meant to stick around for a while :)
But if we remember to adjust the comment when removing the `extends ...` for the next major, this is not a problem